primaryHandler_test.go

```diff
@@ -345,11 +373,31 @@ func TestHandleGetEvents(t *testing.T) {
 					Token: bascule.NewToken("basic", "owner-from-auth", bascule.NewAttributes(
 						map[string]interface{}{})),
 				}
-			case "jwt":
+			case "jwtwithpartners":
```

Comment:
Instead of having `auth` be a string and requiring this switch here, can we make the `auth` in the test cases have the `bascule.Authentication` type? Auths used multiple times can be declared as variables above the `tests` struct creation.

There would be a reason to keep this here if we were using values that change with each test, but it doesn't look like we are to me.